crates/aesgcm/src/lib.rs

```diff
@@ -10,4 +10,4 @@ pub mod wasm_bindgen;
 mod core;
 mod error;
 
-pub use crate::core::aesgcm::{ReExposedAesGcm, BLOCK_LENGTH, KEY_LENGTH, NONCE_LENGTH};
+pub use crate::core::aesgcm::{decrypt, encrypt, BLOCK_LENGTH};
```

Comment:
Why do you need a `BLOC_LENGTH`? It seems to me that this is of no use for a user.

Reply:
indeed, thanks